_posts/2016/2016-05-12-offsite-consulting.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Consulting offsite meeting"
+date: "2016-05-18 16:03:17"
+image: blog-header/consulting-offsite.jpg
+categories: events
+authors: ["Pavlo", "Tobias"]
+---
+
+On 1st of May 2016, the ePages Consulting team set off on a journey to Bad Sulzburg, Germany. After a six-hour drive from Jena, we were welcomed by the nice staff of the Waldhotel, which is situated in a quiet spot of the Black Forest. A good place for us to reflect on what works well in the team and what could be improved.
+
+{% image blog/blog-consulting-offsite-1.jpg %}{% endimage %}
+
+The purpose of this particular trip was to talk about opportunities and challenges faced by our team over time as well as to come up with strategic plans for the future.
+
+## Team building activities
+
+To strengthen the team spirit and escape from office life, we decided to go hiking and enjoy the exceptional diversity and grandiose views of the Black Forest landscape.
+
+{% image blog/blog-consulting-offsite-2.jpg %}{% endimage %}
+
+On the next day after countless meetings we played Kubb to reduce stress and to draw fresh inspiration. Kubb is an exciting team game from Sweden with a very long tradition. The goal of each team is to overturn the wooden figures of the counterparty by throwing wooden sticks. The king with the red crown in the middle of the field must be taken last.
+
+{% image blog/blog-consulting-offsite-3.jpg %}{% endimage %}
+
+## Meetings
+
+During the meetings we worked together on all issues. Each topic had its own presenter. We discussed many different topics, analysed the work from the previous two quarters, and developed a plan to reduce some internal expenses in order to focus on work that creates more profit.
+
+{% image blog/blog-consulting-offsite-4.jpg %}{% endimage %}
+
+On the next day, we had a very important meeting with a new customer. We received feedback on already completed work, and set milestones for the future collaboration. 
+
+{% image blog/blog-consulting-offsite-5.jpg %}{% endimage %}
+
+As the work of the Consulting team is to customise the standard ePages software according to customer-specific requirements, on of our topic was the continuous delivery pipeline. This pipeline should ensure that, each time that a new version is released, the customer-specific adaptations still work and that no side effects occur.
+
+The achieved results such as newly defined Consulting services and a defined workflow to determine business values will bring many benefits but they still need to be reviewed and accepted by our management before they can come into force.
```

Comment:
The achieved results, such as redefined Consulting services and a workflow to determine business values, will bring many benefits for the company, but still need to be reviewed and accepted.